### DISCUSSION TITLE
提示动态无法加载

### DISCUSSION THREAD

**Discussion by WcZaHr** [ywmoyue/biliuwp-lite#339] [General]
### Bug 描述

{"desc":{"user_profile":{"decorate_card":null,"info":{"uid":400919724,"uname":"瑞安航空Ryanair","face":"https://i0.hdslb.com/bfs/face/5c3e2c56487feb6f9c36cd094e98b9d7f59642b2.jpg"},"card":{"official_verify":{"type":-1,"desc":""}},"vip":{"vipType":1,"vipDueDate":1602432000000,"dueRemark":null,"accessStatus":0,"vipStatus":0,"vipStatusWarn":null,"themeType":0},"pendant":{"pid":0,"name":"","image":"","expire":0,"image_enhance":""},"rank":"10000","sign":"bye bye Latehansa","level_info":{"current_level":5,"current_min":0,"current_exp":0,"next_exp":null}},"uid":400919724,"type":1,"rid":"849004671196463110","dynamic_id_str":"849004670968922114","dynamic_id":"849004670968922114","rid_str":"849004671196463110","r_type":0,"bvid":null,"view":0,"repost":0,"comment":0,"like":0,"is_liked":0,"timestamp":1696512704,"pre_dy_id":"849004670971019264","orig_dy_id":"849004670971019264","orig_type":4300,"uid_type":1,"status":1},"extend_json":"{\"from\":{\"from\":\"fastpub.create.share_component\",\"verify\":{\"dc\":{},\"verify_first\":true}}}","card":"{ \"user\": { \"uid\": 400919724, \"uname\": \"瑞安航空Ryanair\", \"face\": \"https:\\/\\/i0.hdslb.com\\/bfs\\/face\\/5c3e2c56487feb6f9c36cd094e98b9d7f59642b2.jpg\" }, \"item\": { \"rp_id\": 849004671196463110, \"uid\": 400919724, \"content\": \"分享动态\", \"ctrl\": \"\", \"orig_dy_id\": 849004670971019264, \"pre_dy_id\": 849004670971019264, \"timestamp\": 0, \"reply\": 0, \"orig_type\": 4300 }, \"origin\": \"{\\\"cover\\\":\\\"https:\\\\\\/\\\\\\/i1.hdslb.com\\\\\\/bfs\\\\\\/archive\\\\\\/6ab7aeb45ae6f3fc33c43151a0c95ee47e3edb22.jpg\\\",\\\"cover_type\\\":2,\\\"fid\\\":23430170,\\\"id\\\":2343017024,\\\"intro\\\":\\\"\\\",\\\"media_count\\\":5,\\\"mid\\\":400919724,\\\"sharable\\\":true,\\\"title\\\":\\\"城市漫步\\\",\\\"type\\\":2,\\\"upper\\\":{\\\"face\\\":\\\"https:\\\\\\/\\\\\\/i0.hdslb.com\\\\\\/bfs\\\\\\/face\\\\\\/5c3e2c56487feb6f9c36cd094e98b9d7f59642b2.jpg\\\",\\\"followed\\\":0,\\\"mid\\\":400919724,\\\"name\\\":\\\"瑞安航空Ryanair\\\"}}\", \"origin_extend_json\": \"{}\", \"origin_user\": { \"info\": { \"uid\": 400919724, \"uname\": \"瑞安航空Ryanair\", \"face\": \"https:\\/\\/i0.hdslb.com\\/bfs\\/face\\/5c3e2c56487feb6f9c36cd094e98b9d7f59642b2.jpg\", \"face_nft\": 0 }, \"card\": { \"official_verify\": { \"type\": -1, \"desc\": \"\" } }, \"vip\": { \"vipType\": 1, \"vipDueDate\": 1602432000000, \"vipStatus\": 0, \"themeType\": 0, \"label\": { \"path\": \"\", \"text\": \"\", \"label_theme\": \"\", \"text_color\": \"\", \"bg_style\": 0, \"bg_color\": \"\", \"border_color\": \"\" }, \"avatar_subscript\": 0, \"nickname_color\": \"\", \"role\": 0, \"avatar_subscript_url\": \"\" }, \"pendant\": { \"pid\": 0, \"name\": \"\", \"image\": \"\", \"expire\": 0, \"image_enhance\": \"\", \"image_enhance_frame\": \"\" }, \"rank\": \"10000\", \"sign\": \"bye bye Latehansa\", \"level_info\": { \"current_level\": 5 } } }","display":{"origin":{"origin":null,"emoji_info":null,"topic_info":null},"emoji_info":null,"topic_info":null}}

### 操作步骤

```plain text
打开，打开动态
```

### 期望结果

正常显示动态

### 截图

<img width="960" alt="image" src="https://github.com/ywmoyue/biliuwp-lite/assets/119170543/18d6ac01-5bf7-4bb5-8a02-ab26be183243">

### 应用来源

Github/其他Github镜像站

### 安装方式

appx直接安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows11 23H2

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
日志能发一下吗
